build.gradle.kts

```diff
@@ -261,6 +288,19 @@ tasks {
     withType<Test>().matching { it.name.startsWith("compatTest") }.configureEach {
         systemProperty("plugin.version", project.version)
     }
+    named<Test>("test").configure {
```

Comment:
This is equivalent:
```suggestion
    test {
```

Reply:
Is that `tasks.test {}` or `project.test {}` just to make sure it's at the right level in the build.gradle.